.goreleaser.yml

```diff
@@ -4,23 +4,27 @@ builds:
 - binary: summon2
   env:
   - CGO_ENABLED=0
+  flags:
```

Comment:
I had to google the meaning `netgo`.  Maybe add a comment:
```
// 'netgo' is a Go build tag for using a pure Go network lib instead of the 
// host's, to produce a statically linked binary
```